mkosi/__init__.py

```diff
@@ -3469,6 +3469,13 @@ def find_files(root: Path) -> Iterator[Path]:
     yield from scandir_recursive(root,
                                  lambda entry: Path(entry.path).relative_to(root))
 
+def cpio_binary() -> str:
+    # Some distros (Mandriva) install BSD cpio as "cpio", hence prefer
+    # GNU "cpio" if it exists, which should be GNU cpio wherever it exists.
+    # We are interested in exposing same behaviour everywhere hence
+    # it's preferable to use the same implementation of cpio
+    # everywhere.
+    return "bsdcpio" if shutil.which("bsdcpio") else "cpio"
 
```

Comment:
this should be two empty lines.

Reply:
removed

Reply:
Sorry, maybe I was unclear. Between function definitions should be two empty lines, so two before and after this function.